cf_xarray/tests/test_accessor.py

```diff
@@ -1233,3 +1233,23 @@ def test_cmip6_attrs():
     )
     assert da.cf.axes["X"] == ["nlon"]
     assert da.cf.axes["Y"] == ["nlat"]
+
+
+def test_custom_criteria():
+    my_custom_criteria = {
+        "ssh": {
+            "standard_name": ("sea_surface_elev*",),
+            "name": ("sea_surface_elevation",),  # variable name
+        },
+    }
+    cf_xarray.accessor.set_options(my_custom_criteria)
+
+    # Match by standard_name regex match
+    ds = xr.Dataset()
+    ds["elev"] = ("dim", np.arange(10), {"standard_name": "sea_surface_elevBLAH"})
+    xr.testing.assert_identical(ds.cf["ssh"], ds["elev"])
+
+    # Match by exact name
+    ds = xr.Dataset()
+    ds["sea_surface_elevation"] = ("dim", np.arange(10))
+    xr.testing.assert_identical(ds.cf["ssh"], ds["sea_surface_elevation"])
```

Comment:
can we adda  second ssh-like variable and test that we get back a dataset for `ds.cf[["ssh"]]` and that it raises and error for `ds.cf["ssh"]`

Reply:
Ok I added these in https://github.com/xarray-contrib/cf-xarray/pull/234/commits/01da6e3aa7081876415a96b8c43600eea870512a